velox/connectors/hive/HiveConnector.h

```diff
@@ -327,4 +327,52 @@ class HiveHadoop2ConnectorFactory : public HiveConnectorFactory {
       : HiveConnectorFactory(kHiveHadoop2ConnectorName) {}
 };
 
+// TODO: deprecate this structure in followup.
```

Comment:
Why not avoid introducing this in the first place?

Reply:
It's used pretty heavily in `PrestoToVeloxQueryPlan`, and given we are moving large piece of code from presto native to here, we decide to do it in next step